cmake/extensions.cmake

```diff
@@ -126,8 +126,29 @@ endfunction()
 # merges the resulting hex files. The 'flash' target is updated to use this
 # merged hex file instead of the 'zephyr.hex' file from the origin context.
 #
-function(zephyr_add_executable name)
-  set_property(GLOBAL PROPERTY IMAGE ${name}_)
+function(zephyr_add_executable name require_build)
+  set(${require_build} 0)
```

Comment:
Strange, is this tested? I would have thought you needed to do set(x y PARENT_SCOPE)

Reply:
we do:)